README_CN.md

```
@@ -41,16 +41,14 @@ python3 -m apps.cli.spider --dry-run    # 预览模式（不写入文件）
 python3 -m apps.cli.pipeline            # 完整流水线（爬虫 + 上传 + 通知）
 ```
 
-完整安装指南请参阅 [本地安装指南](docs/zh/self-hoster/local-setup.md)。
+完整安装指南请参阅 [本地安装指南](docs/handbook/zh/self-hoster/local-setup.md)。
 
 ## 架构
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)